app/views/forms/_screen.html.erb

```diff
@@ -37,8 +37,101 @@
 
     </div>
 
-    <%# questions on the form %>
-    <%= index_table(Questioning, @form.questionings, :within_form => true) if form_mode != :new %>
+    <div class="questions">
+      <input class="btn btn-primary" type="submit" value="Save"></input>
+
+      <h2>Questions</h2>
+
+      <div class="questions-header" id="questions-header">
+        <span><a href="choose-questions"><i class="fa fa-plus"></i> Add Question</a></span>
```

Comment:
Best to use a `link_to` helper here. Also we need to use the translation system. Also I don't think you need the <span>. Try this:
`<a href="<%= choose_questions_form_path(@form) %>"><i class="fa fa-plus"></i> <%= t("form.add_questions") %></a>`.

I know this is a lot to learn!